apps/OpenSpace/ext/launcher/src/profile/timedialog.cpp

```diff
@@ -144,13 +144,13 @@ void TimeDialog::approved() {
     constexpr const int Relative = static_cast<int>(Profile::Time::Type::Relative);
     if (_typeCombo->currentIndex() == Relative) {
         if (_relativeEdit->text().isEmpty()) {
-            _profile.clearTime();
+            _profile.time = std::nullopt;
         }
         else {
             Profile::Time t;
             t.type = Profile::Time::Type::Relative;
             t.value = _relativeEdit->text().toStdString();
-            _profile.setTime(t);
+            _profile.time = std::nullopt;
```

Comment:
Why clear the time here? Is this a copy/paste error?

Reply:
Oops.. it was indeed